test/Microsoft.Extensions.Configuration.UserSecrets.Test/Microsoft.Extensions.Configuration.UserSecrets.Test.xproj

```diff
@@ -14,5 +14,8 @@
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
It's VS fluff that's required for Test Explorer. Other projects have it already..